tests/bsim/bluetooth/mesh/src/test_blob.c

```diff
@@ -1373,11 +1407,214 @@ static void test_cli_fail_on_no_rsp(void)
 	PASS();
 }
 
+#if CONFIG_BT_SETTINGS
+static void cli_stop_setup(void)
+{
+	bt_mesh_device_setup(&prov, &cli_comp);
+
+	(void)target_srv_add(BLOB_CLI_ADDR + 1, true);
+
+	blob_cli_inputs_prepare(BLOB_GROUP_ADDR);
+	blob_cli_xfer.xfer.mode =
+		is_pull_mode ? BT_MESH_BLOB_XFER_MODE_PULL : BT_MESH_BLOB_XFER_MODE_PUSH;
+	blob_cli_xfer.xfer.size = CONFIG_BT_MESH_BLOB_BLOCK_SIZE_MAX * 2;
+	blob_cli_xfer.xfer.id = 1;
+	blob_cli_xfer.xfer.block_size_log = 12;
+	blob_cli_xfer.xfer.chunk_size = 377;
+	blob_cli_xfer.inputs.timeout_base = 10;
+}
+
+static void test_cli_stop(void)
+{
+	int err;
+
+	if (!recover_settings) {
+		settings_test_backend_clear();
+	}
+
+	bt_mesh_test_cfg_set(NULL, 1000);
+	k_sem_init(&blob_caps_sem, 0, 1);
+	k_sem_init(&lost_target_sem, 0, 1);
+	k_sem_init(&blob_cli_end_sem, 0, 1);
+	k_sem_init(&blob_cli_suspend_sem, 0, 1);
+
+	switch (expected_stop_phase) {
+	case BT_MESH_BLOB_XFER_PHASE_INACTIVE:
+		/* There is no use case to stop in Inactive phase */
+		FAIL();
+		break;
```

Comment:
Could this be handled by the default case? Is the default used for any expected cases? If not the BT_MESH_BLOB_XFER_PHASE_INACTIVE case could be removed both here and in the srv test case.

Reply:
yup, makes sens to me